Prog/clean-arlington.Rmd

```diff
@@ -1052,9 +1053,121 @@ rm(mult, other, nunits, cdties)
 
 ```
 
+
+## Add additional variables 
+
+Get additional metric for number of units from the Arlington County address file, which was downloaded in the preclean step.
+
+```{r address-file}
+
+# read address file
+rdir <- paste0("L:/Libraries/RegHsg/Raw/", filepath, "/")
+adfile <- paste0(rdir, filepath, "-address-file.csv")
+address <- read_csv(adfile)
+
+address1 <- address %>% 
+  mutate(address_cap = toupper(FULL_ADDRESS)) %>% 
+  select(address_cap, units_address = UNITCOUNT)
+
+jur5 <- jur4 %>% 
+  mutate(address_cap = toupper(propaddress)) %>% 
```

Comment:
adding obs from jur4 to jur5, okay? 

Also getting:
Duplicated column names deduplicated: '-' => '-_1' [5], 'STATUS' => 'STATUS_1' [21]Parsed with column specification:
cols(
  .default = col_character(),
  `-` = col_logical(),
  Latitude = col_double(),
  Longitude = col_double(),
  Altitude = col_double(),
  `-_1` = col_logical(),
  OBJECTID = col_double(),
  STNUM = col_double(),
  UNITCOUNT = col_double(),
  UPDATE_DATE = col_datetime(format = ""),
  ZIP5 = col_double(),
  STATUS_1 = col_double()
)
See spec(...) for full column specifications.

Reply:
this is just weird column names in the address file, didn't fix it since we only used two columns

Reply:
looking into the increase in observations now

Reply:
there were some observations in the address file that didn't have unique addresses. I collapsed them and summed the units (a lot of them had a value for one of the observations and 0 units for the remaining)